src/core/fonts.js

```diff
@@ -2969,15 +2966,17 @@ class Font {
         // cmap.
         for (const mapping of cmapMappings) {
           let charCode = mapping.charCode;
-          if (
-            cmapPlatformId === 3 &&
-            charCode >= 0xf000 &&
-            charCode <= 0xf0ff
-          ) {
+          if (charCode >= 0xf000 && charCode <= 0xf0ff) {
```

Comment:
Can't we simply reduce the *entire patch* to just the following changes instead, which is less overall duplication?
```suggestion
          if (
            cmapPlatformId === 3 &&
            cmapEncodingId === 0 &&
            charCode >= 0xf000 &&
            charCode <= 0xf0ff
          ) {
```

Reply:
I initially did that but I wasn't happy to add an extra a potentially useless check in the loop and I found it's more consistent to have the platformID, encodingId checks at the same level as the other one. Finally I think the code is more clear like this.